lib/trento_web/controllers/v1/suma_credentials_controller.ex

```diff
@@ -14,7 +16,7 @@ defmodule TrentoWeb.V1.SUMACredentialsController do
     tags: ["Platform"],
     description: "Gets the saved user settings for SUSE Manager",
     responses: [
-      ok: {"The SUSE Manager user settings", "application/json", Schema.SUMACredentials.Settings},
+      ok: {"The SUSE Manager user settings", "application/json", SUMACredentials.Settings},
```

Comment:
Not a blocker, however let's consider sticking to the `SUMA credentials` wording at this level.
(also in the get operation)